### DISCUSSION TITLE
instamine strict mode

### DISCUSSION THREAD

**Discussion by challet** [ConsenSys-archive/ganache#4084] [Q&A] [answered]
Hello, for testing my dapp I'm using ganache with `--miner.instamine=strict` and it seems the transactions are never mined. I thought my dapp had a problem getting the callback but even in metamask they stay as pending. Any idea about what's happening ?

**Comment by davidmurdoch** [comment, marked as answer]
Sometimes this happens because metamask is out is sync with the chain. This can happen after restarting Ganache without also resetting the accounts in metamask. You can reset accounts in the advanced tab of metamask.

**Reply by challet** [reply]
Thanks for the answer. I've also found the `--database.dbPath` option which allows to persist data between Ganache restarts.
